pytr/portfolio.py

```diff
@@ -51,8 +44,7 @@ async def portfolio_loop(self):
 
             if subscription["type"] == "instrument":
                 await self.tr.unsubscribe(subscription_id)
-                pos = subscriptions[subscription_id]
-                subscriptions.pop(subscription_id, None)
+                pos = subscriptions.pop(subscription_id, None)
```

Comment:
Actually, at least typing wise, this leaves a chance for `pos` to be `None`, and changes the current semantic of the code (`NoneType object is not subscriptable`), as opposed to raising a `KeyError`.

```suggestion
                pos = subscriptions.pop(subscription_id)
```

Reply:
Good point. Even better.